R/render-cached-plot.R

```diff
@@ -480,6 +493,7 @@ renderCachedPlot <- function(expr,
         hybrid_chain(possiblyAsyncResult, function(result) {
           width      <- result$width
           height     <- result$height
+          alt     <- result$alt
```

Comment:
(White space formatting to match the lines together)

```suggestion
          height     <- result$height
          alt        <- result$alt
```